fw-child/template/learn/interactive-content.php

```diff
@@ -4,6 +4,22 @@
 
 $interactive = get_field ( 'asset_post_' . $GLOBALS['fw']['current_lang_code'], get_the_ID() );
 
+// Enqueue GSAP and dependencies when id 'timeline' is present in content.
+// Essential for compatibility with old interactive resource post
+function detect_id_and_enqueue_script($content) {
+    $id_to_detect = 'id="timeline"';
+
+    if (strpos($content, $id_to_detect) !== false) {
+		wp_enqueue_script ( 'gsap' );
+		wp_enqueue_script ( 'scrolltrigger' );
```

Comment:
It seems that the _motion.page_ plugin already includes the GSAP and the ScrollTrigger scripts. Could we validate if we really need to re-include them here, else it loads two times each script from different URLs and with different versions.

Maybe we could instead enqueue the _motion.page_'s versions of the scripts (ex: `wp_enqueue_script ( 'mp-gsap-js' );`) ?

Reply:
With the modifications made to the functions.php about registering instead of enqueuing GSAP, they won't load at the same time because of the condition in the `interactive-content.php` file. I think it's a good idea to have GSAP loading independently from the plugin so these pages do not rely on this plugin to work properly. I can change this if you want me to. Let me know.

Reply:
Modifications accepted.